css/styles.scss

```diff
@@ -668,6 +668,11 @@ tr[data-itemtype="PluginFormcreatorCondition"] ~ tr[data-itemtype="PluginFormcre
    background-color: #CCC;
 }
 
+.plugin_formcreator_formTile.tile_template_fix {
```

Comment:
tile_template_fix should not mention the word "fix"

Reply:
done